package.json

```diff
@@ -3,19 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^10.4.9",
     "@testing-library/user-event": "^12.1.3",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "axios": "^0.21.1",
+    "gapi-script": "^1.2.0",
```

Comment:
It's very good that you implemented the calls with axios instead of gapi-script! An alternative would be using `fetch` but axios works great! 
You can remove gapi-script from dependencies